_hooks/pre-commit

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+
+# A git hook script that checks for trailing whitespace
```

Comment:
Ah, I see what you mean with that now, yes, we can do the hooks that way too, I'll fix that.